packages/office-ui-fabric-react/src/components/Keytip/examples/Keytips.Button.Example.tsx

```diff
@@ -2,11 +2,15 @@ import * as React from 'react';
 import { DefaultButton, CompoundButton } from 'office-ui-fabric-react/lib/Button';
 import { keytipMap } from './KeytipSetup';
 import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
 
 export interface IKeytipsButtonExampleState {
   btnDisabled: boolean;
 }
 
+const marginBottom = mergeStyles({ marginBottom: 28 });
```

Comment:
Recommend enabling "Hide whitespace changes" to view this file...the actual changes are:
1. use a Stack rather than `u-marginRight` for horizontal spacing
2. use this button style rather than `u-marginBottom`

Reply:
This seems to be used for `styles` props. Is `mergeStyles` needed? I feel like this instead could be:

```tsx
const marginBottomStyle = { root: { marginBottom: 28 } };
```

and just passed directly as the `styles` prop.

Reply:
Good point. Fixed that.